vm-setup/roles/v1aX_integration_test/tasks/main.yml

```diff
@@ -82,4 +82,8 @@
 
 - name: Node remediation
   include: remediation.yml
-  when: v1aX_integration_test_action == "remediation"
\ No newline at end of file
+  when: v1aX_integration_test_action == "remediation"
+
+- name: test inspection API
+  include: inspection.yml
+  when: v1aX_integration_test_action in "inspection"
```

Comment:
I suppose since it is done before provisioning `inspection` wont require any other script like `verify` ?

Reply:
I think no, because this is happening after `make` is finished, and is suppose to put baremetalhosts in the same state (`Ready`). I added some checks to make sure that baremetalhosts are in Ready state before we move on with provisioning.